op-chain-ops/script/script.go

```diff
@@ -54,6 +55,33 @@ type CallFrame struct {
 	Prank *Prank
 }
 
+type Broadcast struct {
```

Comment:
This might be worth moving to a `broadcast.go` or into `prank.go` (since broadcasts are a type of prank). The `script.go` is getting quite bulky.